app.py

```diff
@@ -58,8 +60,26 @@ def patent_search(patent_title):
     Need to figure out how to implement two word searches, e.g. automotive camera
     """
 
-    return render_template('search_results.html', category=patent_title,
-                           patents=conduct_search(patent_title, limit=10))
+    # Conduct Patent Search
+    patent_results = conduct_search(patent_title, limit=10)
+
+    # Download patents' prior art
+    drawing_files = download_patents(patent_results)
+
+    # Extract images from patents' prior art
+    if drawing_files:
```

Comment:
Add a function to `src.patent_fetcher`, as per comment you can call it `extract_images_from_patent` or something like that

Reply:
done: https://github.com/IshanMi/reverse-image-search-patent/pull/9/files